README.md

```diff
@@ -255,6 +255,7 @@ This repo contains a list of languages that currently compile to or have their V
 ### <a name="ruby"></a>Ruby <sup>[top⇈](#contents)</sup>
 > Ruby is an open source interpreted high-level programming language for general-purpose programming. Created by Matz. Ruby has a design philosophy that emphasizes code readability, notably using as few sigils (special chars`:.{}%[]&=>;`) as possible.
 * [Wruby](https://github.com/pannous/wruby) Web ruby - a port of minimal ruby (mruby).
+* **[run.rb](https://runrb.io/)** ("*runner bee*") allows you to run Ruby code in the browser
```

Comment:
Removing the bold tags and making the url point to the github project just like other entries would be much better. Something like this:

```md
* [run.rb](https://github.com/jasoncharnes/run.rb) run.rb - allows you to run Ruby code in the browser
```